src/librustc_codegen_ssa/back/linker.rs

```diff
@@ -123,6 +123,7 @@ pub trait Linker {
     fn pgo_gen(&mut self);
     fn control_flow_guard(&mut self);
     fn debuginfo(&mut self);
+    fn strip(&mut self);
```

Comment:
```suggestion
    fn strip(&mut self, strip: Strip);
```
It's better not to retrieve session options right in `Linker` methods, `sess` is available there mostly for hacks and very special cases.